examples/pos-mainchain/ecosystem.config.js

```diff
@@ -0,0 +1,34 @@
+const os = require('os');
+const path = require('path');
+const num = 10;
+
+const followers = new Array(num).fill(0).map((_, i) => ({
+	name: `follower_${i}`,
+	script: './bin/run',
+	args: 'start --api-http --api-ws',
+	interpreter: 'node',
+	env: {
+		LISK_LOG_LEVEL: 'debug',
+		LISK_NETWORK: 'alphanet',
+		LISK_PORT: 7667 + i + 1,
+		LISK_API_WS_PORT: 7887 + i + 1,
+		LISK_SEED_PEERS: `127.0.0.1:7667`,
+		LISK_DATA_PATH: path.join(os.homedir(), '.lisk', 'ex-pos-mainchain', `follower_${i}`),
```

Comment:
Don't we want the seed node's data path to also be in the same sub-folder as the followers, instead of the default data path?